### DISCUSSION TITLE
Feature request: iOS App Support Request

### DISCUSSION THREAD

**Discussion by jordanpwood** [FortAwesome/Font-Awesome#20908] [Feature Requests]
### Is your feature request related to a problem?

I'm using Font Awesome Pro in an iOS app. Although it's possible to just import and use the Pro ttf files by themselves, it would mean that I'd have to refer to them by their unicode code, which is hard to read and maintain. There's an open source library [here](https://github.com/LiveUI/Awesome) which I'd like to use. It is mostly a system for auto-generating a mapping of the Font Awesome icon names to their unicode characters. For the Pro icons, it was scraping [this cheatsheet](https://fontawesome.com/cheatsheet) for the icon names. However, that cheatsheet has not been updated for FA 6 icons, so the library can't update to version 6 mappings. 

Is there another place to get the icon name-to-unicode mappings for Pro icons? Hopefully one that will be maintained going forward. Even better would be an officially supported Swift framework for accessing Font Awesome in apps. However, I'd be satisfied if there was a way for that open source project to continue upgrading Font Awesome versions.

### Feature description

Want FontAwesome Pro name to unicode mappings

### Alternatives

No response

### Additional context

No response

### Feature request checklist

 The title starts with "Feature request: " and is followed by the requested feature description
 This is a single feature
 I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sensibleworld** [comment]
@jordanpwood We do have metadata files, such as an icons.yml and .json file that include icon names, along with their supported unicode mappings. Those are available in our icon downloads.
